lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -35,6 +31,25 @@ export class SilentFlowClient extends BaseClient {
      * @param request
      */
     public async acquireToken(request: SilentFlowRequest): Promise<AuthenticationResult> {
+        let result: AuthenticationResult;
+        try {
+            result = this.acquireCachedToken(request);
```

Comment:
Only needs to be `return this.acquireCachedToken(request)`;

Reply:
Done.